WooCommerce/Classes/ViewRelated/Orders/Order Creation/EditableOrderViewModel.swift

```diff
@@ -194,6 +195,20 @@ final class EditableOrderViewModel: ObservableObject {
     ///
     @Published var selectedProductViewModel: ProductInOrderViewModel? = nil
 
+    /// Keeps track of selected/unselected Products, if any
+    ///
+    @Published var selectedProducts: [Product?] = []
+
+    /// Keeps track of selected/unselected Product Variations, if any
+    ///
+    @Published var selectedProductVariations: [ProductVariation?] = []
+
+    /// Keeps track of the total of selected Products and Product Variations, if any
+    ///
+    var selectedItems: [Int64] {
```

Comment:
I would rename this variable to make it more meaningful `selectedItemsProductIDs`, and update the comment accordingly

Reply:
Added some Unit Tests here https://github.com/woocommerce/woocommerce-ios/pull/8943/commits/eca144ab112a22f61c7dfd70f184b51fc4a97b45 and renamed to `selectedProductsAndVariationsIDs` here: https://github.com/woocommerce/woocommerce-ios/pull/8943/commits/08f84066f709be255b272fe6938f6e23770004c0